components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -611,6 +628,17 @@ interface JsonRpcService {
   // Balance is 0x1 if current selected account is the same as token owner's
   // address, otherwise balance is 0x0.
   GetERC721TokenBalance(string contract_address, string token_id, string account_address) => (string balance, ProviderError error, string error_message);
+
+  // Solana JSON RPCs
+  // https://docs.solana.com/developing/clients/jsonrpc-api
+
+  // Returns the balance of the account of provided Pubkey.
+  GetSolanaBalance(string pubkey) =>
+      (uint64 balance, SolanaProviderError error, string error_message);
+  // Returns the token balance of an SPL Token account.
+  GetSPLTokenAccountBalance(string pubkey) =>
+      (string amount, uint8 decimals, string uiAmountString,
+       SolanaProviderError error, string error_message);
```

Comment:
this file is too big already what do you think about move all this to solana.mojom and include that here?

Reply:
I don't see the immediately need here for these two methods, I would prefer keep this PR as is, we can separate Solana out when we see fit a bit later.

Reply:
Agree that we can do this in a future task